data_model/src/query/mod.rs

```diff
@@ -65,6 +64,10 @@ pub trait Query: Into<QueryBox> + seal::Sealed {
     type Output: Into<Value> + TryFrom<Value>;
 }
 
+// TODO: Could we make a variant of `Value` that holds only query results?
+/// Type representing Result of executing a query
+pub type QueryOutput = Value;
```

Comment:
I believe this type can reside in `iroha_core` as well

Reply:
I think I was not the one, who introduced it, but just moved it.

Altough I don't find this alias usefull so I removed it at all

Reply:
fine if you removed it, we're going to have to thing about relationship between queries and expression system anyhow